src/vanilla.jl

```diff
@@ -104,7 +104,7 @@ mutable struct MCTSTree{S,A}
     q::Vector{Float64}
     a_labels::Vector{A}
 
-    _vis_stats::Union{Nothing, Dict{Pair{Int,Int}, Int}} # maps (said=>sid)=>number of transitions. THIS MAY CHANGE IN THE FUTURE
+    _vis_stats::Dict{Pair{Int,Int}, Int} # maps (said=>sid)=>number of transitions. THIS MAY CHANGE IN THE FUTURE
```

Comment:
Is there any possible way that `_vis_stats` can end up being `nothing`? right now it is initialized to a `Dict` and I don't see any code that would change it to `nothing`.

Reply:
I have to say I don't have a perfect understanding of the visualization code, but it seems to me that in some previous version, the `_vis_stats` could be `nothing` and that is when the check `_vis_stats==nothing` was written. Searching the sources, the `_vis_stats` is only ever set in the constructor of `MCTSTree`, so it seems to me the

Reply:
oops, sorry, I misunderstood! I thought the change was the other way around. Can you actually change the error message to something like `"Visualization was not enabled for this MCTS tree or the tree was empty." to be more precise?